fedimg/services/ec2/ec2imguploader.py

```diff
@@ -200,7 +200,8 @@ def _register_image(self, snapshot):
                     virtualization_type=self.image_virtualization_type,
                     architecture=self.image_architecture,
                     block_device_mapping=block_device_map,
-                    root_device_name=root_device_name
+                    root_device_name=root_device_name,
+                    ena_support=True
```

Comment:
Yeah! The plan was to deprecate the PV images due to which I added it directly. I will perform a check for the PV images also so that we maintain the flexibility.

Reply:
PV images are deprecated in #81